frontend_lib/src/component/Timeline/AddFileToUploadButton.jsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types'
 import { translate } from 'react-i18next'
 import PopupUploadFile from '../../container/PopupUploadFile.jsx'
 import IconButton from '../Button/IconButton.jsx'
+import { t } from 'i18next/dist/commonjs'
```

Comment:
this import doesn't seem to be used
and I don't think you're supposed to import anything from i18next, you should only import from 'react-i18next'

Reply:
thanks, it was added automatically, i'll remove it